README.md

```diff
@@ -8,7 +8,7 @@
 The mission of the Financial Desktop Connectivity and Collaboration Consortium (FDC3) is to develop specific protocols and taxonomies to advance the ability of desktop applications in financial workflows to interoperate in a plug-and-play fashion and without prior, bi-lateral agreements.
 
 ## About this Repository
-FDC3 is a standard.  This repository houses the documentation and specifications for the ratified FDC3 standards.
+FDC3 is a standard.  This repository houses the documentation and specifications for the ratified FDC3 standards. To join Working Groups meetings and Mailing lists where the standard is built, please check the [FDC3 space in the FINOS Wiki](http://wiki.finos.org/fdc3).
```

Comment:
This is a handy alternative link that you could also point people to if you like: https://fdc3.finos.org/get-involved